USBPage.qml

```diff
@@ -30,10 +30,9 @@ Item {
     }
 
     ListModel {
-        id: usbModesModel
         //% "Charging only"
         ListElement { title: qsTrId("id-charging-only"); mode: "charging_only" }
-        //% "Adb Mode"
+        //% "ADB Mode"
```

Comment:
Modifying the source code is not enough to update the translations. Could you please also run "lupdate-qt5 asteroid-settings.pro" to regenerate the .ts files ?
Since it is an easy change that should affect all translations equally. The following command should benefit all the translations: sed -i "s@Adb@ADB@" i18n/*.ts